### PR TITLE
Proposal section

### PR DESCRIPTION
**Pull request by boobzzz**

### CODE REVIEW

File: src/partials/proposal.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<section class="section section__proposal">
+  <div class="container container__proposal">
```
Comment (ElephantFrontend) [inline src/partials/proposal.html]: Please don't use container__proposal.
we hav classes .container and .container__content

File: src/partials/proposal.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <h2 class="proposal__header">
+      Master the <span class="proposal__header--span">Language</span> with
+      Our English Package!
+    </h2>
+
+    <div class="proposal__content-wrapper">
+      <div class="proposal__content">
+        <div class="proposal__desc">
+          <h3 class="proposal__subheader">Power of English</h3>
+          <ul class="proposal__feature-list">
```
Comment (ElephantFrontend) [inline src/partials/proposal.html]: please formatted code (spaces)

File: src/partials/proposal.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<section class="section section__proposal">
+  <div class="container container__proposal">
+    <h2 class="proposal__header">
+      Master the <span class="proposal__header--span">Language</span> with
+      Our English Package!
+    </h2>
+
+    <div class="proposal__content-wrapper">
```
Comment (ElephantFrontend) [inline src/partials/proposal.html]: why do you use so many container? Maybe we can use less containers

File: src/css/components/proposal.css [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+.proposal_btn:hover,
+.proposal_btn:focus {
+  background-color: var(--dark-green-color);
+}
+
+.proposal__img {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (ElephantFrontend) [inline src/css/components/proposal.css:125]: please move @media to bottom

File: src/css/components/proposal.css [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+}
+
+.proposal__content-wrapper::before {
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--lighter-green-color);
+  transform: rotate(-8deg);
+  z-index: -1;
```
Comment (ElephantFrontend) [inline src/css/components/proposal.css:26]: i think we have use z-index: -1; for :after and :before

File: src/css/components/proposal.css [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+  }
+
+  .proposal__content {
+    padding: 64px 0 64px 32px;
+  }
+
+  .proposal__header {
+    margin-bottom: 88px;
+    padding-left: 14px;
+    width: 640px;
```
Comment (ElephantFrontend) [inline src/css/components/proposal.css]: We don't need to fix width

File: src/css/components/proposal.css [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+
+  .proposal__feature-list {
+    width: 451px;
+  }
+
+  .proposal__feature {
+    gap: 16px;
+  }
+
+  .proposal__img {
```
Comment (ElephantFrontend) [inline src/css/components/proposal.css:170]: we can use flex for position. (don't use position: absolute and top, right)

File: src/css/components/proposal.css [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+
+@media only screen and (min-width: 1280px) {
+  .container__proposal {
+    padding: 64px 64px 144px 64px;
+  }
+
+  .proposal__content-wrapper::before {
+    top: calc((var(--proposal-front-card-height) - var(--proposal-mid-card-height)) / 2);
+    left: calc((var(--proposal-front-card-width) - var(--proposal-mid-card-width)) / 2);
+    width: 1050px;
```
Comment (ElephantFrontend) [inline src/css/components/proposal.css]: don't need fixed width, height

Comment (boobzzz) [reply src/css/components/proposal.css:189]: According to template, we have different dimensions (width, height) for middle and back cards in desktop layout

File: src/css/components/proposal.css (hunk 5 above)
Comment (ElephantFrontend) [reply src/css/components/proposal.css:26]: fixed

File: src/css/components/proposal.css (hunk 4 above)
Comment (ElephantFrontend) [reply src/css/components/proposal.css:125]: fixed

File: src/css/components/proposal.css (hunk 7 above)
Comment (ElephantFrontend) [reply src/css/components/proposal.css:170]: fixed

**Pull request merged**